### ISSUE TITLE
 File generation of noise.lst

### ISSUE THREAD

**Issue by Sphinxone** [huyanxin/se-cldnn-torch#1]
There is not test in musan file
It is mean that we should use this command to generate noise.lst?
find ${musan}/noise -iname "*.wav" > noise.lst

**Comment by huyanxin** [comment]
sorry, at that time, i use dcase2016' s noise data as test set.
if you want to split some wavs from musan, as test set, you can follow as this:
```bash
nums=500
find ${musan}/noise -iname "*.wav" > noise.lst
shuf nosie.lst > t
head -n ${nums} t > test_noise.lst
tail -n +${nums} t > train_nosie.lst
```
the nums is the nums of test noise clip.

**Comment by Sphinxone** [comment]
thank you!
Sorry for taking your time.
These days i read your code,it's very neat.However,i have some questions about your model.
1.If my understanding is correct,there is no labels in train.lst,and in the project labels is wav file not word.
2.In cldnn.py line128,outputs is three dims,why outputs can feed in nn.Linear which just needs two dims inputs.
3.your model structure is different from cldnn.you use sru first and cnn second,i think it is better than orginal cldnn.Howerver, i can not unserstand why you use mask * inputs?
Recently,i was allocated to do some work about ship recognization using sound of ship.And i tried many ideas,but i found there is little source code about Speech Recognition.Finally,i want to use two ideas,one of them is cldnn,and another is my own ideas which is similiar to Text-CNN.Can i add QQ friend and ask for advice about the detail of cldnn?
